### PR TITLE
added eslint and fix eslint problems

### PR DESCRIPTION
**Pull request by Latyipov**

### CODE REVIEW

File: src/App/components/FactsComponentAdder/FactsComponentAdder.js [hunk 1]
```diff
@@ -25,14 +25,15 @@ export function FactsComponentAdder({ userId }) {
           > add to favorite</button>
         </div>)
     })
-    setFacts(FactsLayout);
+    setFacts(FactsLayout)
   }
 
-  // 
-  return (!facts) ? (
+  //
```
Comment (Dmytraw) [inline src/App/components/FactsComponentAdder/FactsComponentAdder.js:31]: could we remove this line?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      jsx: true
+    },
+    ecmaVersion: 'latest',
+    sourceType: 'module'
+  },
+  plugins: [
+    'react'
+  ],
+  rules: {
+    "react/prop-types": 0,
```
Comment (Dmytraw) [inline .eslintrc.js:27]: i think we can add `no-console` rule too

File: src/App/App.js [hunk 3]
```diff
@@ -1,24 +1,19 @@
...
 
+export function App () {
   return (
     <div>
-      {/* <Link to='signin'> enter</Link>
-      <Link to='main'> main</Link> */}
       <Routes>
         <Route path='signin' element={<EnterPage />} />
         <Route path="/" element={<Navigate to='/main' />} />
         <Route path='/main' element={<MainPage />} />
```
Comment (Dmytraw) [inline src/App/App.js:14]: For the future:

i've noticed that you are using both single(`'`) and double(`"`) quotes. in most cases it's better to use only one type of them.

File: src/App/components/FactsComponentAdder/FactsComponentAdder.js [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 import './FactsComponentAdder.css'
 import { CounterAdder } from '../CounterAdder/CounterAdder'
 import { FactsLoader } from '../FactsLoader/FactsLoader'
 import { PusherToFirebase } from '../FirebaseComponents/PusherToFirebase/PusherToFirebase'
 
-export function FactsComponentAdder({ userId }) {
-  const [factValue, setFactValue] = useState(1);
-  const [facts, setFacts] = useState('');
+export function FactsComponentAdder ({ userId }) {
+  const [factValue, setFactValue] = useState(1)
```
Comment (Dmytraw) [inline src/App/components/FactsComponentAdder/FactsComponentAdder.js:8]: shouldn't this be `0`?
```suggestion
  const [factValue, setFactValue] = useState(0)
```

**Review by Dmytraw — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -11,14 +11,16 @@ In order to show Favorite facts, app uses api that downloads facts for ID.
...
+
+HTML
+CSS
+JavaScript
+React
+Redux
+
+firebase
+API (https://alexwohlbruck.github.io/cat-facts/)
+
```
[the remark this answers is not in the document]
Comment (Latyipov) [reply README.md:23]: added

**Pull request merged**